examples/gh-runner-gke-dind/main.tf

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "runner-gke" {
+  source         = "../../modules/gh-runner-gke"
+  create_network = true
+  project_id     = var.project_id
+  repo_name      = var.repo_name
+  repo_owner     = var.repo_owner
+  repo_url       = var.repo_url
+  gh_token       = var.gh_token
+  machine_type   = "n1-highcpu-4"
+  min_node_count = 3
```

Comment:
Variables for these please

Reply:
This was something I was testing with, didnt intend to be part of the example. Just for some clarity- anything in examples/* is just an example of how to use the actual modules which reside in modules/*. I have used variables with sane defaults for the modules which will be what the end user directly uses.